src/03-modality-agnostic-files.md

```diff
@@ -143,7 +143,7 @@ Example:
 
 ### `README`
 
-Every BIDS dataset SHOULD come with a free form text file (`README`) describing the dataset in more detail.
+A REQUIRED free-form text file, `README` (with an OPTIONAL extension `.md`, `.rst`, or `.txt`), SHOULD describe the dataset in more detail.
```

Comment:
I think we should add a short snippet of text (with links) that these extensions refer to "markdown", "restructured text", and "generic text" (or similar) respectively, and explicitly state that BIDS does not make recommendations with regards to the "flavor" (e.g. Markdown flavor) and does not validate md/rst syntax.

Instead, I think we should RECOMMEND that READMEs are written from a mindset that somebody reading the file in an un-rendered state should still be able to perfectly understand everything.

more opinions by @effigies @yarikoptic perhaps?

Reply:
Shall I simply implement your comments or we're waiting for more feedback?

Reply:
Both are fair approaches, the prior might result in extra work, but also potentially easier/better reviews by the reviewers :-)

Reply:
This kind of makes me think we should add extensions to the schema, so that the description could just link to the glossary instead of describing each of the extensions itself.

Reply:
I don't know what's the rule. Shall I rephrase the text now and add the links later or wait for the glossary to be ready? I could add the links now if I new how they'll be structured.

Reply:
I suggest you add a recommendations along the line of

> Instead, I think we should RECOMMEND that READMEs are written from a mindset that somebody reading the file in an un-rendered state should still be able to perfectly understand everything.

And then we can merge this PR (if other people agree) -- @tsalo (or me) can then add the links in a follow up PR (or in #1063 directly) once #1063 is merged.